docs/index.mdx

```diff
@@ -87,12 +96,18 @@ how to use `traceable()`. See more on the [Annotate code for tracing](./observab
   groupId="client-language"
 />
 
-- View a [sample output trace](https://smith.langchain.com/public/b37ca9b1-60cd-4a2a-817e-3c4e4443fdc0/r).
-- Learn more about tracing in the observability [tutorials](./observability/tutorials), [conceptual guide](./observability/concepts) and [how-to guides](./observability/how_to_guides/index.md).
+Learn more about tracing in the observability [tutorials](./observability/tutorials), [conceptual guide](./observability/concepts) and [how-to guides](./observability/how_to_guides/index.md).
+
+## 5. View your trace
+
+By default, the trace will be logged to the project with the name `default`. You should see the following [sample output trace](https://smith.langchain.com/public/b37ca9b1-60cd-4a2a-817e-3c4e4443fdc0/r) logged using the above code. 
+
+
+## 6. Run your first evaluation
 
-## 5. Run your first evaluation
+[Evaluations](./evaluation/concepts#evaluators) help assess application performance by testing it against a given set of inputs. Evaluations require a system to test, data to serve as test cases, and optionally evaluators to grade the results. 
```

Comment:
the evaluators subheader gets in the weeds of defining evaluator functions, maybe clearer to just point to the concept page as a whole

```suggestion
[Evaluations](./evaluation/concepts) help assess application performance by testing the application against a given set of inputs. Evaluations require a system to test, data to serve as test cases, and evaluators to grade the results. 
```